hs_core/tests/api/rest/test_move_or_rename_file_or_folder.py

```diff
@@ -0,0 +1,85 @@
+import os
+import tempfile
+import zipfile
+
+from rest_framework import status
+
+from hs_core.hydroshare import resource
+from hs_core.tests.api.utils import MyTemporaryUploadedFile
+
+from .base import HSRESTTestCase
+
+
+class TestPublicUnzipEndpoint(HSRESTTestCase):
+    def setUp(self):
+        super(TestPublicUnzipEndpoint, self).setUp()
+
+        self.tmp_dir = tempfile.mkdtemp()
+
+        # Make a text file
+        self.txt_file_name = 'text.txt'
+        self.txt_file_path = os.path.join(self.tmp_dir, self.txt_file_name)
+        txt = open(self.txt_file_path, 'w')
+        txt.write("Hello World\n")
+        txt.close()
+
+        self.raster_file_name = 'cea.tif'
+        self.raster_file_path = 'hs_core/tests/data/cea.tif'
+
+
+        self.rtype = 'GenericResource'
+        self.title = 'My Test resource'
+        res = resource.create_resource(self.rtype,
+                                       self.user,
+                                       self.title,
+                                       unpack_file=False)

```

Comment:
The default for unpack_file is False in create_resource() method definition, so unpack_file=False is not needed here.